src/test/java/com/thirdcc/webapp/web/rest/EventResourceIT.java

```diff
@@ -178,6 +184,21 @@ public void createEvent() throws Exception {
         assertThat(testEvent.getStatus()).isEqualTo(DEFAULT_STATUS);
     }
 
+
+    @Test
+    @Transactional
+    public void createEventWithUserRole() throws Exception {
```

Comment:
can add **shouldThrow403error** behind the method name?

Reply:
@LUXIANZE